torch/nn/modules/transformer.py

```diff
@@ -25,28 +25,29 @@ class Transformer(Module):
         dropout: the dropout value (default=0.1).
         custom_encoder: custom encoder (default=None).
         custom_decoder: custom decoder (default=None).
+        activation: the activation function of encoder/decoder intermediate layer, relu or gelu (default=relu).
 
     Examples::
-        >>> transformer_model = nn.Transformer(src_vocab, tgt_vocab)
-        >>> transformer_model = nn.Transformer(src_vocab, tgt_vocab, nhead=16, num_encoder_layers=12)
+        >>> transformer_model = nn.Transformer()
+        >>> transformer_model = nn.Transformer(nhead=16, num_encoder_layers=12, activation="gelu")
```

Comment:
Let's keep the example simple (a.k.a. without "activation" arg)

Reply:
done